core/tests/protractor/editorAndPlayer.js

```diff
@@ -51,19 +63,19 @@ describe('Full exploration editor', function() {
     editor.saveChanges();
 
     general.moveToPlayer();
-    player.expectContentToMatch(forms.toRichText('this is card 1'));
-    player.submitAnswer('NumericInput', 19);
-    player.submitAnswer('NumericInput', 21);
-    player.expectContentToMatch(forms.toRichText(
+    playerPage.expectContentToMatch(forms.toRichText('this is card 1'));
+    playerPage.submitAnswer('NumericInput', 19);
+    playerPage.submitAnswer('NumericInput', 21);
+    playerPage.expectContentToMatch(forms.toRichText(
```

Comment:
nit: When functions-calls can fit on one line, prefer to let them. So please change this to:

```
playerPage.expectContentToMatch(
  forms.toRichText('this is card 2 with previous answer 21'));
```

here and elsewhere.